conda_build/skeletons/rpm.py

```diff
@@ -61,8 +61,8 @@
 BUILDSH = """\
 #!/bin/bash
 
-mkdir -p ${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot
-pushd ${PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot > /dev/null 2>&1
+mkdir -p ${PREFIX}/{hostmachine}/sysroot
```

Comment:
Is it `host_machine`?

Reply:
No, but maybe it should be, but then we need to double escape things like `{{hostmachine}}` which looks weird, unless one of you python-istas have better ideas?

Reply:
{hostmachine} looks correct given line 520.  `host_machine` is only available in the cdt dictionary.